### PR TITLE
[ci skip] Fix the outdated description for `find_each`.

### PR DESCRIPTION
**Pull request by aditya-kapoor**
Closes https://github.com/rails/rails/issues/33473

### CODE REVIEW

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -368,7 +368,7 @@ end
 
 **`:start`**
 
-By default, records are fetched in ascending order of the primary key, which must be an integer. The `:start` option allows you to configure the first ID of the sequence whenever the lowest ID is not the one you need. This would be useful, for example, if you wanted to resume an interrupted batch process, provided you saved the last processed ID as a checkpoint.
+By default, records are fetched in ascending order of the primary key, which may or may not be an integer. The `:start` option allows you to configure the first ID of the sequence whenever the lowest ID is not the one you need. This would be useful, for example, if you wanted to resume an interrupted batch process, provided you saved the last processed ID as a checkpoint.
```
Comment (kaspth) [inline guides/source/active_record_querying.md]: Let's just remove the `, which …`.

File: guides/source/active_record_querying.md [hunk 2]
```diff
@@ -378,6 +378,14 @@ User.find_each(start: 2000) do |user|
 end
 ```
 
+For example, if you have `emp_id` as the primary_key for the table, to send the newsletters only to users which have emp_id starting from EMP100:
+
+```ruby
+User.find_each(start: 'EMP100') do |user|
+  NewsMailer.weekly(user).deliver_now
+end
+```
```
Comment (kaspth) [inline guides/source/active_record_querying.md]: Don't think this example is needed.

**Comment by aditya-kapoor** [comment]
@kaspth done.

**Comment by kaspth** [comment]
Thanks!

**Pull request merged**